verification/curator-service/ui/src/components/Profile.tsx

```diff
@@ -24,11 +42,310 @@ const styles = makeStyles((theme: Theme) => ({
     },
 }));
 
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
Many of styles declared here are not in use, please avoid copying unnecessary code from other pages as it leads to less readable code